spring-cloud-context/src/main/java/org/springframework/cloud/context/named/NamedContextFactory.java

```diff
@@ -67,6 +67,12 @@ public void setConfigurations(List<C> configurations) {
 		return new HashSet<>(contexts.keySet());
 	}
 
+	protected void createAndCacheContexts() {
```

Comment:
Also, a question @ryanjbaxter, how are such changes which is to fix a downstream issue handled, is it better to open an issue in `spring-cloud-commons` too that shadows the downstream issue and open a PR against that?